AWS/create.yml

```diff
@@ -59,6 +59,38 @@
       delay: 5
       retries: 300
 
+    - name: Get Volumes IDs
+      ec2_vol:
+        instance: '{{ item.instance_ids[0] }}'
+        state: 'list'
+      register: 'ec2_instances_volumes'
+      loop: '{{ instance_create.results }}'
+      loop_control:
+        label: 'ec2-{{ item.instance_ids[0] }}'
```

Comment:
Why do we need label if we're not using it here?

Reply:
Fixed